### ISSUE TITLE
Error 'az find'

### ISSUE THREAD

**Issue by CodeBongBong** [Azure/azure-cli#15591]
## Describe the bug

**Command Name**
`az find`

**Errors:**
```
list index out of range
Traceback (most recent call last):
python3.8/site-packages/knack/cli.py, ln 215, in invoke
    cmd_result = self.invocation.execute(args)
cli/core/commands/__init__.py, ln 579, in execute
    parsed_args = self.parser.parse_args(args)
python3.8/site-packages/knack/parser.py, ln 262, in parse_args
    return super(CLICommandParser, self).parse_args(args)
3.8/lib/python3.8/argparse.py, ln 1771, in parse_args
    self.error(msg % ' '.join(argv))
azure/cli/core/parser.py, ln 171, in error
    recommender = CommandRecommender(*command_arguments, message, cli_ctx)
azure/cli/core/command_recommender.py, ln 61, in __init__
    self.parameters = self._normalize_parameters(parameters)
azure/cli/core/command_recommender.py, ln 257, in _normalize_parameters
    standard_form = sorted_options[0]
IndexError: list index out of range
```

## To Reproduce:
Steps to reproduce the behavior. Note that argument values have been redacted, as they may contain sensitive information.

- _Put any pre-requisite steps here..._
- `az find -q {}`

## Environment Summary
```
macOS-10.15.7-x86_64-i386-64bit
Python 3.8.6
Installer: HOMEBREW

azure-cli 2.13.0

```

**Comment by yungezz** [comment]
hi @mirdaki @christopher-o-toole could you pls help to have a look? thanks

**Comment by yungezz** [comment]
hi @houk-ms could you pls help check also? I found that the error is from the command_recommender.

**Comment by houk-ms** [comment]
Thisi is a positional argument missing issue. Fixed in #15570

**Comment by houk-ms** [comment]
@CodeBongBong Thanks for reporting this issue. PR has been merged to fixed the issue. Close the issue now.
